### PR TITLE
replace collapse with display:none

### LINKED ISSUE #563: remove all references to visibility:collapse
`visibility:collapse` is not supported by gecko yet (at least for non-table elements), and we use it like `display:none` in some places. Let's get rid of that.

### PR DESCRIPTION
**Pull request by paulrouget**
Fix #563

### CODE REVIEW

File: src/browser/suggestion-box.js [hunk 1]
```diff
@@ -132,7 +132,7 @@
       pointerEvents: 'none'
     },
     collapsed: {
-      visibility: 'collapse'
+      display: 'none'
```
Comment (Gozala) [inline src/browser/suggestion-box.js]: This breaks screenshot feature (See https://github.com/mozilla/browser.html/blob/master/src/browser/web-view.js#L354-L357) #266

Comment (Gozala) [inline src/browser/suggestion-box.js]: Maybe `scale(0)` will work, but I have not tried that yet.

Comment (paulrouget) [inline src/browser/suggestion-box.js]: Sorry, I don't understand how that's related.

Comment (Gozala) [inline src/browser/suggestion-box.js]: Never mind I confused this with web-view collapse.

**Pull request merged**